docs/api_reference/comparator.rst

```diff
@@ -0,0 +1,25 @@
+..
+   Copyright (c) 2022, 2024, Oracle and/or its affiliates.
```

Comment:
Shouldn't the copyright be just 2024 since this is new?

Reply:
Yes. Fixed that